File: Recursion/TowerOfHanoi.py
n = 1

def towerOfHanoi(d,A,B,C):
    global n
    if d>0:
        towerOfHanoi(d-1,A,C,B)
        print(f"{n}--> move disk {d} from {A} to {C}")
        n += 1
        print()
        towerOfHanoi(d-1,B,A,C)

d = int(input())

towerOfHanoi(d,'A','B','C')

File: Recursion/TowerOfHanoi.js
var n = 1;

function towerOfHanoi(d,A,B,C){
    if(d>0){
        towerOfHanoi(d-1,A,C,B);
        console.log(`${n++} --> move disk ${d} from ${A} to ${C}`);
        console.log()
        towerOfHanoi(d-1,B,A,C);
    }
}

towerOfHanoi(16,'A','B','C')